examples/supertuxkart/Dockerfile

```diff
@@ -26,7 +26,8 @@ FROM debian:bullseye
 
 RUN apt update && apt-get install -y  wget xz-utils
 
-RUN wget https://github.com/supertuxkart/stk-code/releases/download/1.3/SuperTuxKart-1.3-linux-64bit.tar.xz
+RUN wget --quiet https://github.com/supertuxkart/stk-code/releases/download/1.3/SuperTuxKart-1.3-linux-64bit.tar.xz && \
+tar -xf SuperTuxKart-1.3-linux-64bit.tar.xz && rm -rf SuperTuxKart-1.3-linux-64bit.tar.xz
 
 RUN tar -xf SuperTuxKart-1.3-linux-64bit.tar.xz
```

Comment:
I believe lines 32 & 33 should be removed since you merged them to be part of the previous and that's the reason for the build failure since it can no longer find the file.

Reply:
```suggestion

```